exercises/ex_atm.py

```diff
@@ -0,0 +1,162 @@
+        dict. with user's id as keys,
+        and user's password and balance in a list as values.
+    """
+    try:
+        atm_options = {}
+        with open(file_path, 'r') as file_pointer:
+            for line in file_pointer:
+                line_list = line.rstrip().split(',')
+                atm_options.update({line_list[0]: [line_list[1],
+                                                   float(line_list[2])]})
+
+        return atm_options
+
+    except FileNotFoundError:
+        print(f"Can't access the path to the file you have entered:"
+              f"{file_path}")
+
+
+def get_user_id(atm_info):
+    while True:
+        user_id_input = input("Enter costumer ID, to turn off the ATM enter "
+                              "-1 -> ")
+        if user_id_input == END_ATM_SERVICE:
+            atm_info = None
+            return user_id_input, atm_info
+        elif user_id_input in atm_info:
+            return user_id_input, atm_info
+        else:
+            print("Invalid ID")
+
+
+def atm_service(file_path, atm_options, user_id_input):
+    user_option = input("Enter wanted option, to turn off the ATM enter -1"
+                        + "\nEnter 'options' to see ATM's options. -> ")
+    user_option = user_option.lower()
+    if user_option == END_ATM_SERVICE:
+        update_file_content(file_path, atm_options)
+        atm_options = None
+    elif user_option == "options":
+        print("The ATM allow the following options to a using costumer:" +
+              "\n1. 'check' the balance.\n2. cash 'withdrawal'." +
+              "\n3. cash 'deposit'.\n4. 'change' password.")
+    else:
+        atm_options = check_input(atm_options,
+                                  user_option, user_id_input)
+    return atm_options
+
+
+def check_input(atm_info, user_input, user_id):
+    """Check user option input, and act accordingly if the input is correct.
+
+    Args:
+        atm_info (dict): the dictionary, containing all the atm information.
+        user_input (str): the option choice of the user.
+                          there are four options- check, withdrawal,
+                          deposit and change
+        user_id (str): the id of the user, to find him in the dictionary.
+
+    Return:
+        The new changed dictionary of atm information.
+    """
+    if user_input == "check":
+        print(f"You'r balance: {atm_info[user_id][BALANCE_INDEX]}")
+
+    elif user_input == "withdrawal":
+        atm_info = withdrawal(atm_info, user_id)
+
+    elif user_input == "deposit":
+        atm_info = deposit(atm_info, user_id)
+
+    elif user_input == "change":
+        check_balance(atm_info, user_id)
+
+    else:
+        print("Invalid input")
+
+    return atm_info
+
+
+def withdrawal(atm_info, user_id):
+    """function to withdraw money from atm dict"""
+    withdraw = input("Enter how much you want to withdrawal -> ")
+    if float(withdraw) > 0:
+        atm_info[user_id][BALANCE_INDEX] -= float(withdraw)
```

Comment:
what if I have 0$ in my balance?

Reply:
it will turn to minus-  so if my balance is 0, and i withdraw 20$ my balnace will be -20$.
in the exercise they said the balnce can be negative.